examples/textures/textures_image_loading.c

```diff
@@ -30,6 +30,8 @@ int main(void)
     UnloadImage(image);   // Once image has been converted to texture and uploaded to VRAM, it can be unloaded from RAM
     //---------------------------------------------------------------------------------------
 
+    SetTargetFPS(60);               // Set our game to run at 60 frames-per-second
```

Comment:
To be consistent with all the other examples, this function call should be placed before the points-line above it.

Reply:
Updated.